storage/remote/generic/generic.proto

```diff
@@ -15,9 +15,8 @@ syntax = "proto3";
 
 package generic;
 
-
 message Sample {
-  double value = 1;
```

Comment:
Padding the `=` doesn't seem to be proto formatting standard. I don't think there's one golden one, but two-space indents and no padding is what I see in most official things.